job.model.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose')
+
+exports.jobSchema = new mongoose.Schema({
+    companyId: {
+        type: String,
+        required: true
+    },
+    jobTitle: {
+        type: String,
+        required: true
+    },
+    JD: {

```

Comment:
Avoid acronyms like JD, difficult to understand.

Reply:
OK sir, from now on will make it a habit to use full descriptive words rather than these short code names!